bookCopies/models.py

```diff
@@ -1,10 +0,0 @@
-from django.db import models
-
-from books.models import Book
-
-
-# Create your models here.
-class bookCopies(models.Model):
```

Comment:
class name should not be plural and it's first character should be capital.
Add meta class within this class for the admin

Reply:
Also add helping text + error messages in the fields